css/styles.css

```diff
@@ -0,0 +1,288 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  background: #f4f5f7;
+}
+
+header {
+  background: #f72210;
+}
+
+img {
+  width: 150px;
+}
+
+.cloud {
+  font-size: 16px;
+}
+
+.subscribe {
+  padding: 5px;
+  vertical-align: middle;
+}
+
+.nav-brand-big {
+  width: 350px;
+}
+
+.bar {
+  width: 10px;
+}
+
+.search-form {
+  border: none;
+  width: 85px;
+  height: 20px;
+  margin-top: 15px;
+}
```

Comment:
Please use bootstrap classes when possible. In this case, you can use col-* for the width and change the padding for the height. This will allow you to get a more similar look to the original page. Right now it looks different. Yours:
![Screenshot from 2020-03-14 18-11-02](https://user-images.githubusercontent.com/51926937/76692677-5851da00-661f-11ea-98db-f9c064104d9f.png)

Original:
![Screenshot from 2020-03-14 18-11-57](https://user-images.githubusercontent.com/51926937/76692679-5e47bb00-661f-11ea-8977-248b131760ce.png)

Reply:
made changes as required